email/1.3/modules/ROOT/pages/email-list.adoc

```diff
@@ -1,203 +1,209 @@
-
-The `recent`, `seen`, `deleted` and `answered` parameters will default to INCLUDE. You can combine them however you prefer, however you must always consider that if one of the flags is set as INCLUDE, then the emails marked with that flag will always be included, regardless of the values assigned to the rest of the flags.
-
-For example, if you have the following matcher for your listener:
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:tls="http://www.mulesoft.org/schema/mule/tls" xmlns:email="http://www.mulesoft.org/schema/mule/email"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/email http://www.mulesoft.org/schema/mule/email/current/mule-email.xsd
+http://www.mulesoft.org/schema/mule/tls http://www.mulesoft.org/schema/mule/tls/current/mule-tls.xsd">
+	<email:imap-config name="Email_IMAP" doc:name="Email IMAP" doc:id="e1aa73b1-4719-4463-b0d9-4b92fb6b2622" >
+		<email:imaps-connection host="imap.gmail.com" user="user1@domain.com" password="passwordconnection" >
+			<tls:context enabledProtocols="TLSv1.2,SSLv3">
+				<tls:trust-store path="aTruststore.jks" password="passwordtrust" />
+				<tls:key-store path="aKeystore" password="passwordkey" />
+			</tls:context>
+		</email:imaps-connection>
+	</email:imap-config>
+	<flow name="ListIMAP" >
+		<scheduler doc:name="Scheduler" >
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</scheduler>
+		<email:list-imap doc:name="List - IMAP" config-ref="Email_IMAP"/>
+		<foreach doc:name="For Each" >
+			<set-variable value="payload.body" doc:name="Set Variable" variableName="email-content"/>
+			<set-variable value="payload.attachments.{nameOfTheAttachment}" doc:name="Set Variable" variableName="json-attachment"/>
+		</foreach>
+	</flow>
+</mule>
+----
+
+== List Emails from a POP3 Server
+
+When listing emails, you can filter them using the *Match with* parameter, that defines the criteria to process the emails. All matcher attributes are optional and ignored if they are not provided. They are also related to each other under an `AND` operator, meaning that all the criteria must be true. These attributes vary depending on the type of configuration that you are using. For example, the IMAP protocol provides more metadata about the retrieved email such as the `recent`, `seen`, `deleted`, and `answered` attribute flags.
+
+The following example illustrates how to list all the emails in the configured POP3 mailbox folder that matches with the specified criteria set in the *Match with* parameter for the configuration. The flow initiates with a *Scheduler* component, and then the *List - POP3* operation list only the emails whose subject regex match `BETA`:
```

Comment:
that matches with the >that matches the

Same comment as before about rewording to say "The *Scheduler* component initiates..."

The *List - POP3* operation list > he *List - POP3* operation lists